test/integration_tests/integrationtest_scanner_api.cpp

```diff
@@ -244,6 +324,46 @@ TEST(ScannerAPITests, testReceivingOfMonitoringFrame)
   EXPECT_TRUE(monitoring_frame_barrier.waitTillRelease(DEFAULT_TIMEOUT)) << "Monitoring frame not received";
 }
 
+TEST(ScannerAPITests, shouldNotCallLaserscanCallbackInCaseOfEmptyMonitoringFrame)
+{
+  StrictMock<ScannerMock> scanner_mock;
+  const ScannerConfiguration config{ createScannerConfig() };
+  UserCallbacks cb;
+  ScannerV2 scanner(config, std::bind(&UserCallbacks::LaserScanCallback, &cb, std::placeholders::_1));
+
+  std::vector<double> measures = { 1, 2, 3, 4, 5 };
+  MonitoringFrameMsg msg(TenthOfDegree(0), TenthOfDegree(1), 0, measures);
```

Comment:
Why do we have scanner_mock.sendEmptyMonitoringFrame(); but not something like scanner_mock.sendNormalMonitoringFrame()?

Reply:
Done.